apps/prairielearn/python/python_helper_sympy.py

```diff
@@ -288,9 +298,9 @@ def evaluate(expr: str, locals_for_eval: LocalsForEval) -> sympy.Expr:
     # https://nedbatchelder.com/blog/201206/eval_really_is_dangerous.html
     # http://blog.delroth.net/2013/03/escaping-a-python-sandbox-ndh-2013-quals-writeup/
     #
-    whitelist: ASTWhitelistT = (
-        ast.Module,
-        ast.Expr,
+    # Note some whitelist items were removed. If there are compatibility issues, try
+    # adding those items back.
+    whitelist: ASTWhiteListT = (
```

Comment:
Who is this comment supposed to be here for? 
Most people who use this element are not going to see this is they have problems, they are going to come to pl-help on slack. 

Is there a reason to remove some Whitelist items? If there is a new security problem with some of them, then I'm all for it, but aside from that I don't know that we should remove things.

Reply:
The whitelist for this should be as short as possible. This PR changed the way that the parser works so that certain things that were allowed before no longer need to be for test cases to pass, so I removed the unneeded items.

This comment is basically just a reminder that the removal of items could be the culprit if people run into weird errors. I'm happy to remove the comment if it seems redundant.

Reply:
Ideally ya, the whitelist should be as short as possible, but we are trying to avoid breaking people's questions. 
I don't think we have reason to believe the test cases we have are covering all the cases people are using this element for. 

But anyway, if you want to try to shrink the whitelist, right now is not a bad time since it is summer, just be ready to revert if we have people complain that we broke their questions.

Reply:
I've made comments like these in the past, but going back to them even six months in the future, they've almost always had very little utility. If we're going to keep this comment, I'd like to at least link to this PR so we can easily discover what was removed and when.

```suggestion
    # Note some whitelist items were removed in this PR:
    # https://github.com/PrairieLearn/PrairieLearn/pull/7020
    # If there are compatibility issues, try adding those items back.
    whitelist: ASTWhiteListT = (
```